src/mixins/ui/lasso-tool-ui.js

```diff
@@ -1356,17 +1601,19 @@ export default class LassoButtonGroupController {
       )
     }
 
-    if (lassoToolSetTypes & LassoToolSetTypes.kCircle) {
-      add_handler("circle", CircleShapeHandler)
-    }
-    if (lassoToolSetTypes & LassoToolSetTypes.kPolyLine) {
-      add_handler("polyline", PolylineShapeHandler)
+    const available_lasso_tools = {
```

Comment:
a simpler way to add new tools in the future. Could allow for registering custom tools if that need ever arises.